backend/Server/Server.java

```diff
@@ -0,0 +1,279 @@
+package Server;
+
+import com.sun.net.httpserver.Headers;
+import com.sun.net.httpserver.HttpServer;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.io.File;
+import java.io.OutputStream;
+import java.io.FileInputStream;
+import java.net.InetSocketAddress;
+import java.net.Socket;
+import java.net.URLDecoder;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+// Used for testing to create shallow JSON strings. -Kyle
+import java.lang.StringBuilder;
+
+import static Server.Enum.HttpConstants.*;
+import static Server.Enum.Pages.*;
+
+public class Server implements Runnable {
+    private static String hostname = "localhost"; // Change this if we actually host this on a domain. -Kyle
+    private static final int backlog = -1; // maximum number of connections allowed in queue. >=0 values set value to OS-specific defualts. -Kyle
+    private static final int serverCloseDelay = 2; // how many seconds to wait for connections to close themselves after server stop
+    private static int port = 8081; // Defualt port for http servers is 8080 -Kyle

```

Comment:
inconsistent unless intentional

Reply:
Intentional, I run a personal web server at home so port 8080 is already occupied- changing the port number affects nobody else